### PR TITLE
Fix auto-commenting when publishing updates through `eas update`

### PR DESCRIPTION
**Pull request by floatplane**
This gets PR comments working correctly for EAS update bundles

### CODE REVIEW

**Comment by floatplane** [comment]
## Preview in Expo Go

iOS | Android
--- | ---
[qrcode](exp://u.expo.dev/update/c879c2fe-f6fb-4384-9b84-5f642abb0f81) | [qrcode](exp://u.expo.dev/update/5c3b65d9-90b9-47fc-859b-ae0163f60708)

File: .github/workflows/preview.yaml [hunk 1]
```diff
@@ -39,26 +39,10 @@ jobs:
...
-        run: echo "SELECTED_COLOR=green" >> $GITHUB_OUTPUT
-
-      - name: Get color
-        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
-
-      # - name: 💬 Comment preview
-      #   uses: expo/expo-github-action/preview-comment@v7
-      #   with:
-      #     channel: pr-${{ github.event.number }}
+          ios_id=`echo $output | jq -r '.[] | select(.platform == "ios") | .id'`
```
Comment (stevekuznetsov) [inline .github/workflows/preview.yaml:42]: nit: modern shell-isms would have this as

```sh
echo "ios_id=$( jq --raw-output '.[] | select(.platform == "ios" ) | .id ' <<<"${output}" )" >> "${GITHUB_OUTPUT}"
```

Comment (floatplane) [reply .github/workflows/preview.yaml:42]: sorry, did you just use "modern" and "shell" in the same sentence 😛

Comment (floatplane) [reply .github/workflows/preview.yaml]: TIL about `<<<`

Comment (stevekuznetsov) [reply .github/workflows/preview.yaml]: After putting shell into prod you learn tips and tricks to remove some of the sharp edges

**Pull request merged**